### DISCUSSION TITLE
I lost access to Github Copilot

### DISCUSSION THREAD

**Discussion by cnahmetcn** [community/community#43817] [Copilot Conversations]
### Select Topic Area

General

### Body

I suddenly lost access to Github Copilot today. However, I am still a student. There isn't any mail. Does anyone know about the problem and solve it?
<img width="1334" alt="Screenshot 2023-01-10 at 10 32 05" src="https://user-images.githubusercontent.com/23660881/211489700-d624264c-d22a-4844-91de-0ed778261422.png">

<img width="410" alt="Screenshot 2023-01-10 at 10 36 43" src="https://user-images.githubusercontent.com/23660881/211490041-27e6f171-27cf-4fe6-8e13-7df5ed32f988.png">

**Comment by jongwooo** [comment]
@cnahmetcn See #43634

**Comment by cnahmetcn** [comment]
e-mail arrived. my problem is solved
